modules/ROOT/pages/kubernetes/persistent-volumes.adoc

```diff
@@ -82,6 +83,8 @@ neo4j:
   name: standalone-with-storage-class
 volumes:
   data:
+    labels:
+        data: true
```

Comment:
can we also quote the values "true" ...i believe without quotes the example might just fail...so better to quote the value in documentation

Reply:
But that's a boolean. Are you sure?

Reply:
yeah since its a label...labels are considered strings

Reply:
aah, ok.